src/hotspot/share/code/exceptionHandlerTable.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 1998, 2019, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 1998, 2020, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Should be 2021 now.

Reply:
Thanks for your reminding. Will update it.